easybuild/easyconfigs/f/FreeFem++/FreeFem++-3.58_ebvars.patch

```diff
@@ -0,0 +1,72 @@
+# Pick up EB provided ScaLAPACK and FFTW in PETSc install 
+# August 23rd 2018 by B. Hajgato (Free University Brussels - VUB)
+--- freefem++-3.58/download/ff-petsc/Makefile.orig	2018-01-03 07:55:10.000000000 +0100
++++ freefem++-3.58/download/ff-petsc/Makefile	2018-08-25 13:21:53.409212134 +0200
+@@ -51,7 +51,7 @@
+ ifeq ($(MPICC),$(W_MPICC))
+   WITH_MPI_DIR=
+ else
+-  WITH_MPI_DIR="--with-mpi-dir='$(MPI_DIR)'"
++  WITH_MPI_DIR=--with-mpi-dir='$(MPI_DIR)'
+ endif	
+ DIR_INSTALL_REAL:=$(prefix)/ff-petsc/real
+ DIR_INSTALL_COMPLEX:=$(prefix)/ff-petsc/complex
+@@ -110,14 +110,20 @@
+ 	--with-ssl=0 ---with-x=0 \
+     '--with-scalar-type=real' \
+     $(WITH_MPI_DIR) \
+-	--with-blas-lapack-lib='$(LAPACKLIBS) $(BLASLIBS)' \
+-	--download-scalapack --download-metis --download-ptscotch \
++	--with-blas-lapack-lib="[${BLAS_LAPACK_LIB_DIR}/${BLAS_LAPACK_STATIC_LIBS}]" \
++	--with-scalapack=1 \
++	--with-scalapack-include=${SCALAPACK_INC_DIR} \
++	--with-scalapack-lib="[${SCALAPACK_LIB_DIR}/${SCALAPACK_STATIC_LIBS}]" \
++	--download-metis --download-ptscotch \
+ 	--download-mumps --download-ml --download-hypre --download-parmetis \
+-	--download-superlu --download-pastix --download-suitesparse --download-fftw \
++	--download-superlu --download-suitesparse \
```

Comment:
Why drop `--download-pastix`?

Reply:
`PaStiX` does not work parallel, so it is disabled in the configuration (see now the easyblock). Here, it also breaks the `PETSc` parallel build.